docs/reference/client-libraries.md

```diff
@@ -42,6 +42,7 @@ their authors, not the Kubernetes team.
 | Go       | [github.com/ericchiang/k8s](https://github.com/ericchiang/k8s) |
 | Java (OSGi) | [bitbucket.org/amdatulabs/amdatu-kubernetes](https://bitbucket.org/amdatulabs/amdatu-kubernetes) |
 | Java (Fabric8, OSGi) | [github.com/fabric8io/kubernetes-client](https://github.com/fabric8io/kubernetes-client) |
+| Node.js  |  (https://github.com/Goyoo/node-k8s-client) |
```

Comment:
this renders parantheses around the URL and won't make it clickable on k8s.io website. Can you follow the same syntax as the other lines?